Components/input.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles.css" />
+    <link rel="stylesheet" href="/utility.css" />
+    <link rel="stylesheet" href="/variables.css" />
+    <link rel="stylesheet" href="avatar.css" />
+    <link rel="stylesheet" href="badge.css" />
+    <link rel="stylesheet" href="input.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Input</title>
+  </head>
+  <body>
+    <nav class="flex space-between">
+      <div class="nav-brand">Water Style UI</div>
+      <div>Documentation</div>
+    </nav>
+    <div class="content-wrapper flex">
+      <aside class="sidebar flex flex-column gap-l">
+        <a href="/index.html">Home</a>
+        <p>Components</p>
+        <ul class="flex flex-column gap-xl">
+          <li><a href="avatar.html">Avatar</a></li>
+          <li><a href="badge.html">Badge</a></li>
+          <li><a href="alert.html">Alert</a></li>
+          <li><a href="button.html">Buttons</a></li>
+          <li><a href="card.html">Cards</a></li>
+          <li><a href="input.html" class="active-link">Input</a></li>
+          <li><a href="image.html">Image</a></li>
+        </ul>
+      </aside>
+
+      <div class="flex flex-column main gap-m debug">
+        <h1>Input Text Box</h1>
+
+        <div class="container flex flex-column gap-m">
+          <label for="demo" class="input-label">Label</label>
+          <input
+            type="text"
+            id="demo"
+            class="input-field"
+            placeholder="Please Enter Text Here"
+          />
+        </div>
+
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%2859%2C130%2C246%2C1%29&t=seti&wt=none&l=auto&width=503&ds=true&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=10.5px&lh=142%25&si=false&es=2x&wm=false&code=%253Clabel%2520for%253D%2522demo%2522%2520class%253D%2522input-label%2522%253ELabel%253C%252Flabel%253E%250A%253Cinput%2520type%2520%253D%2520%2522text%2522%2520id%2520%253D%2520%2522demo%2522%2520class%2520%253D%2520%2522input-field%2522%2520placeholder%253D%2522Please%2520Enter%2520Text%2520Here%2522%253E"
+          style="
+            width: 503px;
+            height: 71px;
+            border: 0;
+            transform: scale(1);
+            overflow: hidden;
```

Comment:
don't use inline styles for iframes just add a class in util i think .

Reply:
Hey, this i-frame was directly copied from carbon.now.sh, but yeah it makes sense to put it in a class maybe